plugins/casutg/src/main/java/com/blazemeter/jmeter/threads/AdditionalFieldsPanel.java

```diff
@@ -70,6 +74,7 @@ public void UItoModel(AbstractDynamicThreadGroup tg, JMeterVariableEvaluator eva
     public void clearUI() {
         logFile.setText("");
         iterations.setText("");
+        sameUserBox.setSelected(true);
```

Comment:
It it really the default behavior in JMeter to enable it?

Reply:
JMeter TheadGroup has it enabled by default.
https://github.com/apache/jmeter/blob/ab65ec18103fad25cbd85aa05078bb7f89c5caca/src/core/src/main/kotlin/org/apache/jmeter/threads/AbstractThreadGroupSchema.kt#L44

These clearUI use the default values ​​but the model always wins.
What you made me see is that I do not actually have the default value set to true in the model when the property does not exist, now I replicated the same as JMeter's ThreadGroup.
Thanks for the question, I'm looking to make that fix to imitate the same behavior as JMeter's ThreadGroup.